spec/osut_tests_spec.rb

```diff
@@ -367,29 +367,87 @@
 
   it "checks construction thickness" do
     translator = OpenStudio::OSVersion::VersionTranslator.new
+    version    = OpenStudio.openStudioVersion.split(".").join.to_i
     expect(cls1.clean!).to eq(DBG)
 
+    # The v1.11.5 (2016) seb.osm, shipped with OpenStudio, holds (what would now
+    # be considered as deprecated) a definition of plenum floors (i.e. ceiling
+    # tiles) generating several warnings with more recent OpenStudio versions.
```

Comment:
Harmonized with (more detailed) [fixes](https://github.com/rd2/tbd_tests/pull/32) introduced in **TBD Tests**.